bin/soar-stack.sh

```diff
@@ -61,6 +61,39 @@ create_secrets() {
     wait_build "${SECRETS_STACK_NAME}"
 }
 
+
+
+# WAF stack deployment
+export WAF_TEMPLATE="${REPO_ROOT}/services/WAF/templates/aws-waf-security-automations.template"
+export WAF_ALB_CONFIG="${REPO_ROOT}/services/WAF/templates/waf-api-deploy.toml"
+export WAF_CLOUDFRONT_CONFIG="${REPO_ROOT}/services/WAF/templates/waf-api-deploy.toml"
+
+create-waf-stack()  {
+    echo "Do you wish to deploy to endpoint ALB?"
+    select yn in "Yes" "No"; do
+    case $yn in
+        Yes ) sam deploy \
+        --template-file file://"${WAF_TEMPLATE}" \
```

Comment:
`--stack-name` is missing

![image](https://user-images.githubusercontent.com/14272872/96451657-a6892480-1263-11eb-8666-97e56c22f5e5.png)